Backend/dev/migrations/mock-registry/0000-init.sql

```diff
@@ -0,0 +1,2 @@
+-- duplicated from seed, app is not running without migration
+ALTER TABLE atlas_registry.subscriber OWNER TO atlas_registry_user;
```

Comment:
This also is it required to be pushed ?

Reply:
This is also required for local testing. Mock-registry failed to run locally because of empty directory `dev/migrations/mock-registry.` 
If it bothers us, I can remove

Reply:
Ohh yes, someone faced this issue too, we just created a directory to fix it. Sure we can have this local fixes to solve such issues